hedera-node/hedera-mono-service/src/jmh/java/com/hedera/node/app/service/mono/setup/Constructables.java

```diff
@@ -92,14 +89,9 @@ public static void registerForContractStorage() throws ConstructableRegistryExce
         ConstructableRegistry.getInstance()
                 .registerConstructable(
                         new ClassConstructorPair(ContractKeySerializer.class, ContractKeySerializer::new));
-        ConstructableRegistry.getInstance()
-                .registerConstructable(new ClassConstructorPair(ContractKeySupplier.class, ContractKeySupplier::new));
         ConstructableRegistry.getInstance()
                 .registerConstructable(
                         new ClassConstructorPair(IterableContractValue.class, IterableContractValue::new));
-        ConstructableRegistry.getInstance()
-                .registerConstructable(new ClassConstructorPair(
-                        IterableContractValueSupplier.class, IterableContractValueSupplier::new));
     }
```

Comment:
Were these intentional deletions? They don't look directly related to jasper db

Reply:
Yes, they are. They are quite closely related. The thing is that MerkeDB has slightly different approach to serialization and all the `*Supplier` classes are no longer used, so I deleted them all.

Reply:
Yes, I saw more of how this relates further down. Thanks!